platform/web/os_web.cpp

```diff
@@ -174,7 +174,10 @@ void OS_Web::add_frame_delay(bool p_can_draw) {
 #endif
 }
 
-void OS_Web::vibrate_handheld(int p_duration_ms) {
+void OS_Web::vibrate_handheld(int p_duration_ms, float amplitude) {
+	if (amplitude != 1.0) {
+		WARN_PRINT("Cannot change amplitude on web build!");
```

Comment:
```suggestion
		WARN_PRINT_ONCE("Cannot change amplitude on web build!");
```
Just a suggestion, might be too much to warn constantly